ui/App.svelte

```diff
@@ -43,6 +44,11 @@
   import ModalCollect from "./Modal/Funding/Pool/Collect.svelte";
   import UserProfile from "./Screen/UserProfile.svelte";
 
+  // We receive the backend address from index.ts before the app loads and
+  // svelte-spa-router can mess with `window.location`.
+  export let backendAddress = "localhost:17246";
```

Comment:
I don't understand the relation between this value and its comment above, could you elaborate?

Reply:
Is this description clearer?
https://github.com/radicle-dev/radicle-upstream/pull/1492/commits/fa20392eb28f7b53d4ef4712a8c6350b61c0b42a

Reply:
Ya super clear :ok_hand: